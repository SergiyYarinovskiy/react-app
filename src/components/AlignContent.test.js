import React from 'react';
import renderer from 'react-test-renderer';
import Component from './AlignContent';

it('renders without crashing', () => {
  const tree = renderer.create(<Component/>).toJSON();
  expect(tree).toMatchSnapshot();
});
