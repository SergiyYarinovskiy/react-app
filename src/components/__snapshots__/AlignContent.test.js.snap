// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <h3>
    align-content - 
    :
</h3>,
  <div
    className={undefined}
>
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "20px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "40px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "60px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "80px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "20px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "40px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "60px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "80px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "20px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "40px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "60px",
              }
        }
    />
    <div
        className="rectangle-horizontal"
        style={
            Object {
                "width": "80px",
              }
        }
    />
</div>,
]
`;
